archai/nlp/nvidia_transformer_xl/wt103_base.yaml

```diff
@@ -122,7 +122,7 @@ dgx1_1gpu_fp32: &dgx1_1gpu_fp32
 toy: &toy
    train:
       <<: *train
-      dataset: wt2
+      dataset: wt103
```

Comment:
This is only for toy mode. I can explain this in next meeting.

Reply:
I have removed the commit with such modification from the PR.